app/plugin.go

```diff
@@ -146,6 +148,11 @@ func (a *App) InitPlugins(pluginDir, webappPluginDir string) {
 	}
 	a.SetPluginsEnvironment(env)
 
+	if synchErr := a.SynchPlugins(); synchErr != nil {
+		mlog.Error("Failed to synch plugins with filestore", mlog.Err(synchErr))
+	}
+
+	//Prepackaged
```

Comment:
nit: I don't think this commend adds any value

Reply:
Agreed. Slipped in from my pseudocode 😑